### PR TITLE
testboston: prepare auth0 pages for i18n

### PR DESCRIPTION
**Pull request by yufengwng**
Updating testboston's auth0 pages to make it easier to do i18n.

### CODE REVIEW

File: study-builder/tenants/testboston/pages/login.html [hunk 1]
```diff
@@ -84,37 +84,66 @@
...
+      },
+      fields: {
+        firstNamePlaceholder: "Your first name",
+        lastNamePlaceholder: "Your last name",
+        nonBlankHint: "Can't be blank"
+      },
+      lastLoginInstructions: 'Last time you signed in with',
+      message: {
+        alreadyRegisteredText: 'If you have already registered, sign in here.',
+        helpHtml: 'For help, please contact ' + studyHelpEmail + '.',
```
Comment (yufengwng) [inline study-builder/tenants/testboston/pages/login.html:110]: This one is concatenated with other HTML and put into `innerHTML` so I named it `helpHtml`.

File: study-builder/tenants/testboston/pages/login.html [hunk 2]
```diff
@@ -84,37 +84,66 @@
...
+        loginLinkText: 'Sign in with existing account.',
+        registerMessageHtml: "If you haven't registered yet, please <a class='auth0-lock-alternative-link' href='" + registerUrl + "'>do so here</a>.",
```
Comment (fat23cat) [inline study-builder/tenants/testboston/pages/login.html:112]:
```suggestion
        registerMessageHtml: "If you haven't registered yet, please <a class='auth0-lock-alternative-link' href='" + registerUrl + "/prism" + "'>do so here</a>.",
```

**Review by fat23cat — COMMENTED** [review]
Overall looks good, only one suggestion.

File: study-builder/tenants/testboston/pages/login.html [hunk 3]
```diff
@@ -84,37 +84,66 @@
...
-    var title = 'Welcome!';
-    if (mode === 'login') {
-        title = 'Please Sign In';
-    }
-    else if (mode === 'signup') {
-        title = 'Create your account';
-    }
-
     var loginUrl = config.callbackURL + '/login';
     var registerUrl = '##BASE_URL##';
```
Comment (fat23cat) [inline study-builder/tenants/testboston/pages/login.html:88]:
```suggestion
    var registerUrl = '##BASE_URL##' + '/join';
```

Comment (yufengwng) [reply study-builder/tenants/testboston/pages/login.html:88]: 🤔 Hmmm, this URL has been pointing folks back to the homepage and no one has complained about it so far. Adding `/join` will change the user experience. Maybe we should hold off on that change until either Jen/Erin/Kiara asks for it?

Comment (fat23cat) [reply study-builder/tenants/testboston/pages/login.html:88]: Okay

**Review by fat23cat — APPROVED** [review]

**Pull request merged**